<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>suggestset</class>
 <widget class="QWidget" name="suggestset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>242</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>词语联想设置</string>
  </property>
  <widget class="QLabel" name="label_syllable_location">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>170</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>从第几个音节开始联想</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_suggestWord_location">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>170</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>联想词在候选词中的位置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_suggest_word_count">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>170</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>联想词个数</string>
   </property>
  </widget>
  <widget class="CustomizeCombobox" name="comboBox_syllable_location">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>86</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>86</width>
     <height>25</height>
    </size>
   </property>
   <property name="maxCount">
    <number>9</number>
   </property>
  </widget>
  <widget class="CustomizeCombobox" name="comboBox_suggestWord_location">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>86</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>86</width>
     <height>25</height>
    </size>
   </property>
   <property name="maxCount">
    <number>9</number>
   </property>
  </widget>
  <widget class="CustomizeCombobox" name="comboBox_suggest_word_count">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>86</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>86</width>
     <height>25</height>
    </size>
   </property>
   <property name="maxCount">
    <number>9</number>
   </property>
  </widget>
  <widget class="QPushButton" name="button_reset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>195</y>
     <width>89</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>89</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>89</width>
     <height>32</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>恢复默认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_ok">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>195</y>
     <width>89</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>89</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>89</width>
     <height>32</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cancel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>195</y>
     <width>89</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>89</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>89</width>
     <height>32</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>298</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>298</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>298</width>
     <height>32</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="title_label">
        <property name="text">
         <string>联想配置</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="clostBtn">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomizeCombobox</class>
   <extends>QComboBox</extends>
   <header location="global">customize_ui/customizecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
